v2/invoke.go

```diff
@@ -83,6 +80,12 @@ func invoke(ctx context.Context, call APICall, settings CallSettings, sp sleeper
 		if strings.Contains(err.Error(), "x509: certificate signed by unknown authority") {
 			return err
 		}
+		if apierr, ok := FromError(err); ok {
+			return apierr
+		}
+		if settings.Retry == nil {
+			return err
+		}
```

Comment:
Could double check with double if is ok to move down after checking `strings.Contains`
@noahdietz

Reply:
Seems fine to me.